CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+Contributing guidelines
+==
+
+Format :clap: your :clap: sources :clap: before :clap: pushing
+
+    sbt fmt
+
+and use [Scalazzi subset of scala](https://github.com/scalaz/scalazzi) using
```

Comment:
I like this contributing guide. We may want to add details on how to actually get these things setup if it's not clear from their readmes.

Reply:
Created a separate issue to deal with this: #3